mu-plugins/blocks/global-header-footer/postcss/header/header.pcss

```diff
@@ -90,11 +90,6 @@ html {
 	}
 }
 
-/* The classic themes don't use the same modal view, so the offset needs to be changed. */
```

Comment:
It's possible this is still necessary after all - I've tested on wp.org/about and wp.org and this works, but when I tried wp.org/support the X was missing again. I'd like to merge & deploy this to fix the desktop issue, though.